guides/main_concepts/indexes.md

```diff
@@ -64,16 +64,39 @@ For example, on the following search query: `the great gatsby`, if the presence
 
 [More information about stop-words](/guides/advanced_guides/stop_words.md)
 
-## Rules of fields
+## Fields properties
 
 In MeiliSearch, by default, every document field is searchable and returned on search queries.
 
-You could give a list of **indexed fields** that will be added by MeiliSearch on document addition — making it easy to ignore some uninteresting field that you have not cleaned out.
+The properties of the fields can be changed in the [settings](/references/settings.md).
 
-You could give a list of **searchable fields** that will be used to determine the relevancy of the documents - excluding some fields with non-relevant information.
+Fields can be either :
+- Searchable
+- Displayed
+- Stored
```

Comment:
It could not be not stored, It's perhaps preferable to not talk about ir

Reply:
When is it not stored? AcceptNewFields doesn't store them?

Reply:
If `AccetNewFields=false` then, new fields will not be searchable or displayed. But they will be stored on the  KVStore

Reply:
Ah yes, I misread your sentence. Did you saw the part where I explained the `stored`? Just a little lower.

Reply:
I think it's not essential to talk about this. It seems to be an obvious behavior. Probably not obvious for us internally because we have discussed that.

Reply:
I removed it from the bullet point and changed the title of the Stored attributes part. Do you accept it this way?